### PR TITLE
Add breaking change to CHANGELOG

### PR DESCRIPTION
**Pull request by andygup**
This update adds a breaking change notice for the removal of the default setting for `config.assetsPath`.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ The `next` version of 4.19 is now available.  Planned release date is April 2021
 
 ## Breaking changes
 
-* n/a 
+* Local NPM installs of `@arcgis/core` and `arcgis-js-api` are now required to set `config.assetsPath`, the default value has been removed. Errors will be thrown if this property is not manually set.
```
Comment (dasa) [inline CHANGELOG.md:9]:
```suggestion
* Local NPM installs of `@arcgis/core` and `arcgis-js-api` are now required to set `config.assetsPath`. The default value has been removed. Errors will be thrown if this property is not set. 
```

**Review by dasa — APPROVED** [review]

**Comment by steveoh** [comment]
What was the default path?

**Comment by andygup** [comment]
@steveoh Here's the default path as documented for 4.18: https://developers.arcgis.com/javascript/latest/api-reference/esri-config.html#assetsPath

**Comment by steveoh** [comment]
Thanks! That might be a good addition to the change log. It would help me understand what style it is be expecting so I can make the changes necessary.

**Comment by andygup** [comment]
Sure, we can add an example.

**Comment by steveoh** [comment]
Maybe it's an upgrade guide item but it's so tiny it _can_ fit in the change log

**Pull request merged**